docker-entrypoint.sh

```diff
@@ -126,19 +126,22 @@ if [ "$1" = 'rabbitmq-server' ]; then
 			].
 		EOF
 	fi
+fi
 
-	if [ "$ssl" ]; then
-		# Create combined cert
-		cat "$RABBITMQ_SSL_CERT_FILE" "$RABBITMQ_SSL_KEY_FILE" > /tmp/combined.pem
-		chmod 0400 /tmp/combined.pem
+if [ "$ssl" ]; then
+	# Create combined cert
+	# If calling this script with an exec this is problematic, as it needs to be owned 400 for perms but this code can run as root OR rabbitmq.
+	# As a workaround prefix the combined cert with the username so each user only operates on the file they control.
+	combined_file="/tmp/$(id -un)_combined.pem"
```

Comment:
But then that opens up the root/non-root problem again unless we also add a chmod to ensure the file its owned by rabbitmq:rabbitmq and rely on the root user being able to read any file.  A chmod can be added but I'm not a huge fan of running as rabbitmq sometimes and root other, I'm a fan of always being rabbitmq as simpler behavior.  I'm at a loss as to what workflow might be broken by not being root, though I can certainly appreciate not changing things unnecessarily.